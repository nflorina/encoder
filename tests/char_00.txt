g
(
m
d
~
o
?
I
i
,
(
1
END
