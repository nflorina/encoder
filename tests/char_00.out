0 11 0
gg(ggm(gdmg~dgo~g?ogI?giIg,ig(,
gg(ggm(gdmg~dgo~g?ogI?giIg,ig(,
